api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -145,7 +129,7 @@ default void setDiskEnvironmentType(Disk disk, @Nullable Object diskLabels) {
   LeonardoListRuntimeResponse toListRuntimeResponse(LeonardoGetRuntimeResponse runtime);
 
   @Nullable
-  @Named("cloudContextToGoogleProject")
+  @Named("legacy_cloudContextToGoogleProject")
```

Comment:
Are the legacy mappings needed?

Reply:
yes, for legacy AppsApi, RuntimesApi, etc